### PR TITLE
Image gallery

### PR DESCRIPTION
**Pull request by psteyer**

### CODE REVIEW

**Comment by psteyer** [comment]
on commit 29c83cb having issues pulling API data correctly.

Getting an unsafe port error.

**Comment by psteyer** [comment]
Adding @cjmartian to review merges and make sure nothing breaks any further after restructuring the project

**Comment by psteyer** [comment]
Changed port value to a new port. Port originally used was a reserved port.

File: API/.env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+API_KEY=552737677385964
```
Comment (cjmartian) [inline API/.env]: Cycle your keys now, you should never commit credentials like this

**Review by cjmartian — CHANGES_REQUESTED** [review]

**Comment by cjmartian** [comment]
This is impossible to review. Over `350,000` changes on `2420` files. 

I think what's happened here is you've committed some files that aren't actually your code. Maybe some nodejs files or something. Either way, we should setup a `.gitignore` to not let this happen again. You should only ever commit your actual code; setup should be handled by whatever env manager you use.

**Comment by psteyer** [comment]
Closing pull request and retrying on branch `image-gallery-second-attempt`

**Pull request closed without merging**